### PR TITLE
add case study for unionbigdata in English

### PR DESCRIPTION
**Pull request by pandaoknight**
What type of PR is this?

/kind documentation

What does this PR do / why do we need it:

This PR adds an English case study titled "UnionBigData's Utilization of Karmada in the Construction of I3Plat at BOE."

Which issue(s) does this PR fix:
Fixes #none

Special notes for your reviewer:

The first two pictures of the architecture have been redrawn in both English and Chinese. The last five pictures remain unchanged. May readers understand them by referring to the context.

I have generated and reviewed the English translation myself. I believe it is understandable, and the terms are correct.

### CODE REVIEW

**Comment by pandaoknight** [comment]
I noticed the DCO problems, I'll fix it right now with new signed-off-by line.

File: docs/casestudies/unionbigdata.md [hunk 1]
```diff
@@ -1,95 +1,89 @@
 ---
-title: 数之联在 BOE 的 I3Plat 建设中使用 Karmada
+title: UnionBigData's Utilization of Karmada in the Construction of I3Plat at BOE
 ---
 
-## 行业背景
+## 1. Industry Background
```
Comment (windsonsea) [inline docs/casestudies/unionbigdata.md:5]:
```suggestion
## Industry Background
```
Not very necessary to maintain both `##` and `1.`, which have the same meaning.

File: docs/casestudies/unionbigdata.md [hunk 2]
```diff
@@ -1,95 +1,89 @@
...
 
-通常我们用 Service 来暴露 Java 实现的功能，并用 Java Fabric8 等客户端调用 kubectl get --raw 来实现调用：
+Typically we expose functions implemented in Java using Service, and invoke kubectl get --raw with Java Fabric8 or similar clients:
 
 ```
 /apis/cluster.karmada.io/v1alpha1/clusters/%s/proxy/api/v1/namespaces/%s/services/%s/proxy/%s
 ```
 
-#### 1、集群监控
+#### 2.2.1. Cluster Monitoring
```
Comment (windsonsea) [inline docs/casestudies/unionbigdata.md:46]:
```suggestion
#### Cluster Monitoring
```
ditto. It need to change more than your expected if you add or remove sections later

**Review by windsonsea — COMMENTED** [review]
A few concerns about headings

File: docs/casestudies/unionbigdata.md (hunk 1 above)
Comment (pandaoknight) [reply docs/casestudies/unionbigdata.md:5]: I'll remove the numbers.

File: docs/casestudies/unionbigdata.md (hunk 2 above)
Comment (pandaoknight) [reply docs/casestudies/unionbigdata.md:46]: Got it, I've remove all the numbers. Should I add a new commit or squash them into one?

Comment (windsonsea) [reply docs/casestudies/unionbigdata.md:46]: Seems all right. It has only one commit now.

Comment (pandaoknight) [reply docs/casestudies/unionbigdata.md]: > Seems all right. It has only one commit now.

Thanks for reviewing.

**Review by RainbowMango — APPROVED** [review]
/lgtm
/approve

Thanks @pandaoknight @windsonsea ~

**Pull request merged**